src/pages/App.tsx

```diff
@@ -75,6 +75,10 @@ const HeaderWrapper = styled.div<{ scrolledState?: boolean; nftFlagEnabled?: boo
   width: 100%;
   justify-content: space-between;
   position: fixed;
+  transition: ${({ theme, nftFlagEnabled }) =>
+    nftFlagEnabled &&
```

Comment:
i don't think this is what we want? doesn't this mean when this is released this will happen for tokens too?

Reply:
yeah we want to remove it for all pages (tokens, nft, everything) in phase 1: https://uniswaplabs.atlassian.net/browse/WEB-1565